tmt/utils.py

```diff
@@ -47,6 +47,7 @@
 if TYPE_CHECKING:
     import tmt.base
     import tmt.cli
+    import tmt.result
```

Comment:
Probably introduced by the previous code changes that are not happening anymore, this import is no longer needed.

Reply:
Deleting this import.